docker-compose.yml

```
@@ -58,7 +59,7 @@ services:
     ]
     volumes:
       - ./wp-data:/docker-entrypoint-initdb.d
-      - db_data:/var/lib/mysql
+      - ./db_data:/var/lib/mysql
```

```suggestion
      - db_data:/var/lib/mysql
```

This should use the `db_data` volumen that is defined bellow